docs/layer_configuration.md

```diff
@@ -0,0 +1,342 @@
+
+# Profile Name
+# =====================
+# Name of the profile specified by the profile file to use. If set to
+# ${VP_DEFAULT}, the layer will load the first profile in the file.
+khronos_profiles.profile_name = VP_KHR_roadmap_2022
+
+# Schema Validation
+# =====================
+# Validate the profile file against the Vulkan SDK profile schema if the file is
+# found.
+khronos_profiles.profile_validation = false
+
+# Simulate Profile Capabilities
+# =====================
+# Control of the simulated capabilities of the Vulkan physical device from the
+# selected Vulkan Profile.
+khronos_profiles.simulate_capabilities = SIMULATE_API_VERSION_BIT,SIMULATE_FEATURES_BIT,SIMULATE_PROPERTIES_BIT
+
+# Unspecified Features
+# =====================
+# Feature values when not specified in the select Vulkan profiles.
+khronos_profiles.default_feature_values = DEFAULT_FEATURE_VALUES_DEVICE
+
+# Debug Actions
+# =====================
+# This indicates what action is to be taken when a layer wants to report
+# information
+khronos_profiles.debug_actions = DEBUG_ACTION_STDOUT_BIT
+
+# Message Types
+# =====================
+# This is a comma-delineated list of options telling the layer what types of
+# messages it should report back
+khronos_profiles.debug_reports = DEBUG_REPORT_WARNING_BIT,DEBUG_REPORT_ERROR_BIT,DEBUG_REPORT_NOTIFICATION_BIT
+```
+
+#### Layer Settings File location on Linux and macOS
+
+Unix systems store the layer setting file in the following path:
+
+- `$HOME/.local/share/vulkan/settings.d/vk_layer_settings.txt`
+
+#### Layer Settings File location on Windows
+
+Windows systems store the layer setting file in the following path:
+
+- `%HOME%\AppData\Local\LunarG\vkconfig\override\vk_layer_settings.txt`
+
+## Configuring Vulkan Layers using Environment Variables
+
+### Finding Vulkan Layers
+
+In order to enable a Vulkan layer from the command-line, you must first make sure:
+
+  1. The layer's Manifest JSON file is found by the Vulkan Desktop Loader because it is in:
+      * One of the standard operating system install paths
+      * It was added using one of the layer path environment variables (`VK_LAYER_PATH` or `VK_ADD_LAYER_PATH`).
+      * See the `Layer Discovery` section of the Vulkan Loader's [Layer Interface doc](https://github.com/KhronosGroup/Vulkan-Loader/blob/main/docs/LoaderLayerInterface.md).
+  2. The layer's library file is able to be loaded by the Vulkan Desktop Loader because it is in:
+      * A standard library path for the operating system
+      * The library path has been updated using an operating system-specific mechanism such as:
+          * Linux: adding the path to the layer's library .so with `LD_LIBRARY_PATH`
+          * MacOS: adding the path to the layer's library .dylib with `DYLD_LIBRARY_PATH` 
+  3. The layer's library file is compiled for the same target and bitdepth (32 vs 64) as the application
```

Comment:
bitdepth -> bitness

Reply:
Are you sure about this? This is from the original file

Reply:
not sure. bitness is more common for arch and bitdepth is for color, but since it was used in original documentation I guess that’s correct too.